crates/web-plugins/oob-messages/src/plugin.rs

```diff
@@ -53,17 +48,13 @@ impl Plugin for OOBMessages {
         let env = get_env()?;
         let mut fs = StdFileSystem;
 
-        let oob_inv = retrieve_or_generate_oob_inv(
-            &mut fs,
-            &env.server_public_domain,
-            &env.server_local_port,
-            &env.storage_dirpath,
-        )
-        .map_err(|err| {
-            PluginError::InitError(format!(
-                "Error retrieving or generating OOB invitation: {err}"
-            ))
-        })?;
+        let oob_inv =
+            retrieve_or_generate_oob_inv(&mut fs, &env.server_public_domain, &env.storage_dirpath)
```

Comment:
You removed the server_local_port ?

Reply:
yes all this should be part of the SERVER_PUBLIC_DOMAIN as in prod we won't this anymore